src/styles/styles.js

```diff
@@ -718,24 +717,22 @@ const styles = {
 
     textInputContainer: {
         flex: 1,
-        borderRadius: variables.componentBorderRadiusNormal,
         justifyContent: 'center',
         height: '100%',
-        backgroundColor: themeColors.componentBG,
-        borderWidth: 1,
+        backgroundColor: 'transparent',
+        borderBottomWidth: 2,
```

Comment:
Following bugzero checklist: I think this change caused this: https://github.com/Expensify/App/issues/14632

Thanks @situchan for pointing that out.

Reply:
Makes sense :)